test/count.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
why not a python script?

Reply:
this is to mimic how users are actually going to use it: a bash job script that runs something else